libs/langgraph-cua/src/types.ts

```diff
@@ -128,9 +173,14 @@ export function getConfigurationWithDefaults(
   config: LangGraphRunnableConfig
 ): typeof CUAConfigurable.State {
   return {
+    provider: config.configurable?.provider ?? "scrapybara",
     scrapybaraApiKey:
       config.configurable?.scrapybaraApiKey ||
       getEnvironmentVariable("SCRAPYBARA_API_KEY"),
+    hyperbrowserApiKey:
+      config.configurable?.hyperbrowserApiKey ||
+      getEnvironmentVariable("HYPERBROWSER_API_KEY"),
+    sessionParams: config.configurable?.sessionParams ?? {},
```

Comment:
default to undefined if not set
```suggestion
    sessionParams: config.configurable?.sessionParams ?? undefined,
```